### PR TITLE
[8.0] l10n_lu_mis_reports

### PR DESCRIPTION
**Pull request by sbidoul**
This is a data module with the Luxembourg P&L and Balance sheet in mis_builder format.
- [x] This PR depends on OCA/account-financial-reporting#86

### CODE REVIEW

File: l10n_lu_mis_reports/data/mis_report_abr_pl.xml [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="default_css_style">background-color: orange; font-weight: bold</field>
+      <field name="sequence">10</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_abr_pl_fr430">
+      <field name="report_id" ref="mis_report_abr_pl"/>
+      <field name="name">fr430</field>
+      <field name="description">1. à 2. Charges brutes (moins B.1 à B.4)</field>
+      <field name="expression">max(balp[60%,61%,70%,71%,72%,73%], 0)</field>
```
Comment (laetitia-gangloff) [inline l10n_lu_mis_reports/data/mis_report_abr_pl.xml]: I get the following error :

title="fr430 = max(balp[60%,61%,70%,71%,72%,74%], 0)

Traceback (most recent call last):
  File "/home/lga/project/odoo/addons-account-financial-reporting/mis_builder/models/mis_builder.py", line 527, in _compute
    kpi_val = safe_eval(kpi_eval_expression, localdict)
  File "/home/lga/project/odoo/parts/odoo/openerp/tools/safe_eval.py", line 314, in safe_eval
    return eval(c, globals_dict, locals_dict)
  File "", line 1, in <module>
ValueError: "_max() takes exactly 1 argument (2 given)" while evaluating
u'max((0.0), 0)'"

Not sure it is the right way, but I replaced with :
balp[60%,61%,70%,71%,72%,74%] > 0 and balp[60%,61%,70%,71%,72%,74%] or 0

**Comment by sbidoul** [comment]
@laetitia-gangloff can you try with https://github.com/OCA/account-financial-reporting/pull/116

**Comment by sbidoul** [comment]
or `max([balp[60%,61%,70%,71%,72%,74%], 0])`

File: l10n_lu_mis_reports/data/mis_report_abr_pl.xml [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="default_css_style">font-weight: bold</field>
+      <field name="sequence">325</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_abr_pl_fr461">
+      <field name="report_id" ref="mis_report_abr_pl"/>
+      <field name="name">fr461</field>
+      <field name="description">RESULTAT DE L'EXERCICE A TRANSFERER AU 142</field>
+      <field name="expression">-balp[resultat,resultat-transfert]</field>
```
Comment (adrienpeiffer) [inline l10n_lu_mis_reports/data/mis_report_abr_pl.xml:429]: Where resultat and transfert variable are defined ?

Comment (laetitia-gangloff) [inline l10n_lu_mis_reports/data/mis_report_abr_pl.xml:429]: https://github.com/odoo/odoo/blob/8.0/addons/l10n_lu/account.account.template-2011.csv#L586
https://github.com/odoo/odoo/blob/8.0/addons/l10n_lu/account.account.template-2011.csv#L1161

**Comment by sbidoul** [comment]
Apparently, most contributors who have the technical knowledge to review this in details have contributed to this PR and approve. So it's going to be difficult to have reviewers here.

This is running fine in production in several instances.

So if there is no objection, I'll allow myself an exception to the official rule and merge this.

**Comment by adrienpeiffer** [comment]
:+1: I think this one is now ready to merged

**Pull request merged**